ChineseFilter/sections/specific.txt

```diff
@@ -43,6 +43,7 @@ $script,xmlhttprequest,third-party,domain=00ksw.com|01zww.com|0biquge.com|114zw.
 !
 ! NOTE: Specific rules
 !
+eprice.com.hk##.text-center:has(> div[class^="ad-"] > ins.adsbygoogle)
```

Comment:
As far as I understand, this rule is required on the main page - `https://www.eprice.com.hk/`
It looks like that element with `adsbygoogle` class is added after a delay (probably once a specific request is loaded, if I'm not wrong), so there is a moment where this element is not hidden.

https://github.com/user-attachments/assets/e3e1a853-ae37-43a1-a83b-97d79a2b86ad

Please change it to something like:
```suggestion
eprice.com.hk##.main-section > .text-center[style^="min-height:"]:has(> div[class^="ad-"])
```
This way it should be hidden faster.
If it's also required in another place, then please add both rules.